api/src/app.ts

```diff
@@ -1,21 +1,21 @@
 import express from "express";
 import morgan from "morgan";
 import { logStream } from "./services/logging";
+import { Client as ElasticClient } from "@elastic/elasticsearch";
 import {
   errorHandler,
   articlesController,
   articleController,
 } from "./controllers";
 import { Config } from "../config";
-import { Clients } from "./types";
 
-const createApp = (clients: Clients, config: Config) => {
```

Comment:
Not especially important, but I would personally not have changed this - if we needed to access another external service at some point, we'd have to reintroduce the generic `Clients` as this has coupled us to an `app` using one single `ElasticClient`.